config/locales/codelists/2_03/iati.en.yml

```diff
@@ -286,24 +286,30 @@ en:
       ZA: South Africa
       ZM: Zambia
       ZW: Zimbabwe
+      'N/A': Not Applicable
```

Comment:
I'm not sure this should be listed as country, if it's not a valid country value in most contexts. Instead, this could be a switch on this specific step to determine whether intended beneficiaries are required or not.

Reply:
I know this shouldn't be there. The reason I put it in there is because of the way the `activity_presenter` works. If it wasn't there, when you are in the `activity details` page on an activity that has "Not Applicable" as `intended beneficiaries` you will not see the "Not Applicable" value. You'll see something like `translation missing: en.iati....etc` I couldn't find another way to display this value in the activity details without putting it on that file/list